src/helpers/getPackageVersion.js

```diff
@@ -31,7 +32,7 @@ module.exports.getAngularVersion = getAngularVersion
 const getAngularRuntimeVersion = async function (root) {
   let packagePath
   try {
-    const siteRequire = createRequire(root)
+    const siteRequire = createRequire(join(root, ':internal:'))
```

Comment:
the previous version ended up not working when actually installed and resulted in MODULE_NOT_FOUND (that was not immediate apparent, because we swallow the error message)